### PR TITLE
Add accordion module

### PR DESCRIPTION
**Pull request by ckeisato**

### CODE REVIEW

File: src/accordion/Accordion.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+/**
+ * @overview An Accordion module comprised of sections that toggle open and close
+ * @module Accordion.js
+*/
+
+export const Accordion = {
+
+  accordionID: ``,
```
Comment (misscs) [inline src/accordion/Accordion.js]: What if it is a class? ID feels very specific. Maybe something like accordingEl?

File: src/accordion/_base.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   * @module Accordion
+   * @overview
+   * @@@ todo: standardize height transition, also make proportional to max-height/media queries
+ **/
+
+.accordion-list-item {
+   margin: ms(1) 0;
+ }
+
+.accordion-section {
```
Comment (misscs) [inline src/accordion/_base.scss]: Curious to why we need to nest classes in this instance?

File: src/accordion/_base.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    cursor: pointer;
+
+    [data-icon="arrow"] {
+      transform: rotate(90deg);
+    }
+  }
+
+  &.is-open {
+    .accordion-section-list {
+      max-height: 1000px;
```
Comment (misscs) [inline src/accordion/_base.scss]: This feels dangerous. Why do we need to set max-height?

File: src/accordion/_base.scss [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+      @include respond-at-and-above($break-medium) {
+        max-height: 500px;
```
Comment (misscs) [inline src/accordion/_base.scss]: Same comment here. Where is this value coming from?

File: src/accordion/_base.scss (hunk 3 above)
Comment (ckeisato) [inline src/accordion/_base.scss]: @misscs This isn't ideal, but we don't have a good way to animate height at the moment.  These values would also have to be reset depending on the content for each accordion.

File: src/accordion/Accordion.js (hunk 1 above)
Comment (ckeisato) [inline src/accordion/Accordion.js]: This is amended.

File: src/accordion/_base.scss (hunk 2 above)
Comment (ckeisato) [inline src/accordion/_base.scss]: This is amended.

**Pull request merged**